sources/packages/backend/apps/db-migrations/src/sql/Applications/Add-application-edit-status-columns.sql

```diff
@@ -0,0 +1,57 @@
+ALTER TABLE
+  sims.applications
+ADD
+  -- Adding new column with the default 'Edited' to later be removed.
+  COLUMN application_edit_status sims.application_edit_status NOT NULL DEFAULT 'Edited',
+ADD
+  -- Default will be dropped at the end of this script.
+  COLUMN application_edit_status_updated_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+ADD
+  -- A NOT NULL constraint will be added at the end of this script.
+  COLUMN application_edit_status_updated_by INT REFERENCES sims.users(id),
+ADD
+  COLUMN application_edit_status_note_id INT REFERENCES sims.notes(id);
+
+COMMENT ON COLUMN sims.applications.application_edit_status IS 'Status of the application related to possible editions that resulted in different versions of the same application.';
+
+COMMENT ON COLUMN sims.applications.application_edit_status_updated_on IS 'Last date and time when the edit status changed.';
+
+COMMENT ON COLUMN sims.applications.application_edit_status_updated_by IS 'User that changed the edit status last time.';
+
+COMMENT ON COLUMN sims.applications.application_edit_status_note_id IS 'Note added by the Ministry while approving or declining the edited application.';
+
+-- First ever saved application should be considered the original one.
+-- The submission will make the parent_application_id and the preceding_application_id the same from its own ID.
+UPDATE
+  sims.applications
+SET
+  application_edit_status = 'Original'
+WHERE
+  id = parent_application_id
```

Comment:
`id = parent_application_id` - Will this alone not sufficient to identify `Original` edits?

Reply:
Good point, yes it will. I will change it.

Reply:
Changed.